dev/Common/Converters.cpp

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <common.h>
+#include "Converters.h"
+
+CppWinRTActivatableClassWithBasicFactory(CornerRadiusFilterConverter)
+
+CornerRadiusFilterConverter::CornerRadiusFilterConverter()
+{
+}
+
+winrt::IInspectable CornerRadiusFilterConverter::Convert(
+    winrt::IInspectable const& value,
+    winrt::TypeName const& targetType,
+    winrt::IInspectable const& parameter,
+    winrt::hstring const& language)
+{
+    auto radius = unbox_value<winrt::CornerRadius>(value);
+    auto filter = unbox_value<winrt::hstring>(parameter);
```

Comment:
Is it possible to make this an enum? Or does that make things more complicated to author in the markup?

Reply:
I just tried but looks like the auto string-enum conversion only works in style setters, and using the enum type directly in xaml didn't seem to work either.

I've added an invalid arg exception to help enforce the values.